migrations/versions/d79024110f49_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: d79024110f49
+Revises: db7f5df23e1b
+Create Date: 2022-01-07 16:52:28.448119
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd79024110f49'
+down_revision = 'db7f5df23e1b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('systems', sa.Column('release_version', sa.String(length=100), nullable=True))
```

Comment:
```suggestion
import sqlalchemy as sa
from sqlalchemy.dialects.postgresql import JSONB

# revision identifiers, used by Alembic.
revision = 'd79024110f49'
down_revision = 'db7f5df23e1b'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('systems', sa.Column('operating_system',  postgresql.JSONB(astext_type=sa.Text()), nullable=True))
```

For reference - https://github.com/RedHatInsights/inventory-schemas/blob/master/schemas/system_profile/v1.yaml#L252

Reply:
Changes :rocket: